dlc_developer_config.toml

```diff
@@ -27,11 +27,17 @@ trcomp_mode = false
 # Please only set it to True if you are preparing a Benchmark related PR
 # Do remember to revert it back to False before merging any PR (including Benchmark dedicated PR)
 benchmark_mode = false
+# Set deep_canary_mode to true to simulate Deep Canary Test conditions on PR for all frameworks in the
+# build_frameworks list below. This will cause all image builds and non-deep-canary tests on the PR to be skipped,
+# regardless of whether they are enabled or disabled below.
+# Set graviton_mode to true to run Deep Canaries on Graviton images.
+# Do remember to revert it back to false before merging any PR.
+deep_canary_mode = true
 
 [build]
 # Add in frameworks you would like to build. By default, builds are disabled unless you specify building an image.
 # available frameworks - ["autogluon", "huggingface_tensorflow", "huggingface_pytorch", "huggingface_tensorflow_trcomp", "huggingface_pytorch_trcomp", "pytorch_trcomp", "tensorflow", "mxnet", "pytorch", "stabilityai_pytorch"]
-build_frameworks = []
+build_frameworks = ["tensorflow", "pytorch", "autogluon", "mxnet"]
```

Comment:
Believe we want build_framework list empty for this initial as the intention is to make this the new toml template.

Reply:
I've put a random collection of frameworks in the `build_frameworks` list to showcase to reviewers what the PR experience would look like.

For tensorflow and pytorch, the **dlc-pr-deep-canary-test** jobs run the newly added simple deep canary test. For others, the **dlc-pr-deep-canary-test** job completes with a no-op.